### PR TITLE
added jax.numpy.pad function

### PR DESCRIPTION
**Pull request by x0axz**
Close https://github.com/unifyai/ivy/issues/11779

### CODE REVIEW

**Comment by x0axz** [comment]
@zaeemansari70 please review this PR.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py [hunk 1]
```diff
@@ -1453,6 +1454,97 @@ def test_jax_numpy_row_stack(
...
+@handle_frontend_test(
+    fn_tree="jax.numpy.pad",
+    dtype_and_input_and_other=_pad_helper(),
+    reflect_type=st.sampled_from(["even", "odd"]),
+    test_with_out=st.just(False),
+)
+def test_jax_numpy_pad(
+    *,
+    dtype_and_input_and_other,
+    reflect_type,
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py:1514]: unused argument, was this by mistake? 🤔

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

Your PR looks good, torch and tensorflow are currently failing but thats because of an unrelated problem with `scatter_nd`.
Just 1 thing I pointed out on the specific line.

Thanks

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py (hunk 1 above)
Comment (x0axz) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py:1514]: yes

**Comment by x0axz** [comment]
@sherry30 Thanks for the review. Also, if you could, please review this PR too, as I am not getting any response from the assignee. https://github.com/unifyai/ivy/pull/13864
Thanks

**Review by sherry30 — APPROVED** [review]
lgtm!

**Pull request merged**